docs/release/release_0_4_1.md

```diff
@@ -87,6 +88,9 @@ investigate some crashes that it seemed to be contributing to. See #1905.
   Instead, use ``viewer.grid.enabled = <True/False>``. (#1821)
 - ``Viewer.grid_stride`` and ``Viewer.grid_size`` are deprecated. Instead,
   use ``Viewer.grid.stride`` and ``Viewer.grid.shape``. (#1821, #1847)
+- Use ``Viewer.grid.events.update`` is removed, there are now
+  individual events for each attribute. Use ``Viewer.grid.events.connect``
+  to connect with all of them. (#1926)
```

Comment:
```suggestion
```

This was never released, right? So it is not an API change.

Reply:
ok that's fair, i was just being conservative - but good point